### DISCUSSION TITLE
Redux-Toolkit Init?

### DISCUSSION THREAD

**Discussion by LOGESH-B** [gbowne1/codebooker#106] [Ideas]
Implementation of the Redux-Toolkit solves the major thing in the Frontend.Makes Frontend flow to next level
- [ ] Making store makes easy access of values

Any Comments?
I am asking this question because we are not having that much nested components

**Comment by gbowne1** [comment]
I think we should just use useReducer, Context API, etc. and custom hooks and make use of what react gives us out of the box before we add more stuff to the project especially when theres still quite a bit to add.

Oh and this is better as a discussion in Discussion tab

If it turns out we need more stuff then am happy implementing redux and react-redux
